tests/jobs/localisation/test_config1.py

```diff
@@ -0,0 +1,831 @@
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE1:PARAM2",
+                "PARAM_NODE2:PARAM1",
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE2:PARAM3",
+            ],
+        ),
+        (
+            ["P*2*", "PARAM_NODE3*"],
+            [
+                "PARAM_FIELD2",
+                "PARAM_GEN:2",
+                "PARAM_NODE1:PARAM2",
+                "PARAM_NODE1X:X2",
+                "PARAM_NODE22:P1",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE2:PARAM1",
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:Y1",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE2Y:Y3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_SURFACE2",
+            ],
+        ),
+        (
+            ["P*2:*", "PARAM_NODE3*"],
+            [
+                "PARAM_NODE2:PARAM1",
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE22:P1",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE3:PARAM4",
+            ],
+        ),
+        (
+            "PARAM_NODE1:PARAM1",
+            ["PARAM_NODE1:PARAM1"],
+        ),
+        (
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X*",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X2",
+                "PARAM_NODE1X:X3",
+            ],
+        ),
+        (
+            ["PARAM_NODE3:P*", "PARAM_NODE2*:*2*"],
+            [
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE3:PARAM4",
+            ],
+        ),
+        (
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:*",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE1X:X1",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:Y1",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE2Y:Y3",
+            ],
+        ),
+        (
+            "PARAM_FIELD*",
+            ["PARAM_FIELD1", "PARAM_FIELD2", "PARAM_FIELD3"],
+        ),
+        (
+            "PARAM_FIELD1",
+            ["PARAM_FIELD1"],
+        ),
+        (
+            "PARAM_GEN:*",
+            ["PARAM_GEN:0", "PARAM_GEN:1", "PARAM_GEN:2", "PARAM_GEN:3", "PARAM_GEN:4"],
+        ),
+    ],
+)
```

Comment:
Are all of these testing something unique, or is there a lot of repetition here? If there is duplication, a simplification would be good.

Reply:
I have removed a few that does not contribute with more than those that still remains.